ApolloExtra.lua

```diff
@@ -2469,67 +2591,153 @@ OlympusLootData.ApolloUpgrade = {
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ZeusFirstPickUp", "PoseidonFirstPickUp", "AthenaFirstPickUp", "AphroditeFirstPickUp", "AresFirstPickUp", "ArtemisFirstPickUp", "DionysusFirstPickUp", "ApolloGift01", "ZeusGift01", "ArtemisGift01", "ApolloFirstPickUp" },
+				{ Cue = "/VO/Apollo_0084",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "I am Zeus' son, Zagzag, but Hera is not my mother. Her name is Leto. **Pause** My mother I mean, she taught Artemis and I everything we know, Sometimes I wish she was in Olympus with us, but she doesn't go well with Hera." },
+			},
+			ApolloAboutLeto02 =
+			{
+                Name = "ApolloAboutLeto02",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ApolloAboutLeto01", "ApolloFirstPickUp" },
+				{ Cue = "/VO/Apollo_0085",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "Leto gave birth to us on an island. Delos was the name. She was fleeing a Python, but don't worry, I defeated the mighty beast and now she's free to go anywhere... Except Olympus, of course..." },
+			},
+			ApolloAboutArtemisLeto01 =
+			{
+                Name = "ApolloAboutArtemisLeto01",
+				PlayOnce = true,
+				RequiredFalseTextLinesThisRun = GameData.GodAboutGodVoiceLines,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				RequiredTextLines = { "ApolloAboutLeto01", "ApolloAboutLeto02", "ApolloFirstPickUp" },
+				RequiredGodLoot = "ArtemisUpgrade",
+				{ Cue = "/VO/Apollo_0086",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
+					Text = "My sister doesn't talk much about our mother, Leto. I guess it's just how she is, not a very vocal character when it isn't about hunting something... But, I am sure you have somehow notice. Am I right, Zagzag." },
 			},
-
 			ApolloPostGiftPickup01 =
 			{
                 Name = "ApolloPostGiftPickup01",
 				PlayOnce = true,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
 				RequiredTextLines = { "ApolloGift01", },
-				{ Cue = "/VO/Apollo_0040",
-					Emote = "PortraitEmoteAffection",
+				{ Cue = "/VO/Apollo_0087",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "How I {#DialogueItalicFormat}love {#PreviousFormat}to have these unexpected little run-ins with you, love! You are just so spontaneous, and I've a liking for that sort of thing!" },
+					Text = "I don't know why, but you are in my mind a lot, lately. I just have this feeling I must meet you somehow... This here should help make it happen." },
```

Comment:
"I don't know why, but you've been in my mind a lot, lately. I just have this feeling I must meet you somehow... This should help make it happen."